### ISSUE TITLE
Bodymovin doesn't export wll my animation. Only empty demo who can't reproduce in lottie player

### ISSUE THREAD

**Issue by saravaldes** [airbnb/lottie-web#1932]
Hello,
I'm a UX designer and kind off beginner in After Effects.

I created an animation with AE, and Bodymovin doesn't preview or properly render the animation into the json file. I am trying to use the animation for an iOS app. When I try to display the animation in the lottie player, none of my vectors show.

Attached animation .aep file and screenshots. I've read a lot of posts here and other forums but didn't find a solution.

https://drive.google.com/open?id=1T0o3ou_vsH-Md9CO2fs6lm7aatqp3aIE

![Captura de pantalla 2020-01-09 a las 16 23 02](https://user-images.githubusercontent.com/58988115/72080052-5e180000-32fc-11ea-9e1f-8928351ae7a6.png)

![Captura de pantalla 2020-01-09 a las 16 22 36](https://user-images.githubusercontent.com/58988115/72080072-63754a80-32fc-11ea-8b38-7c2f821abbf4.png)

**Comment by Danjuanlab** [comment]
Right-click your AI layers and convert them into shapes so you can export them.

**Comment by saravaldes** [comment]
Thanks a lot! But when I convert the layers into shapes, some layers get dark and the background gets grey. Do you know why? Sorry, I'm new in AE :-((

**Comment by Danjuanlab** [comment]
> Thanks a lot! But when I convert the layers into shapes, some layers get dark and the background gets grey. Do you know why? Sorry, I'm new in AE :-((

Some production methods in illustrations cannot be directly converted to shapes, mesh gradients, these effects can be produced in AI, but they cannot be achieved in AE, so when you convert them, you will lose color and other information
